gpu_monitor.py

```diff
@@ -238,6 +278,14 @@ def main(argv):
             error('Could not open server file {}'.format(args.server_file))
             return
 
+    try:
```

Comment:
This should be under `if args.taskset` so the file is only read when requested by the user